packages/block-library/src/template-part/index.php

```diff
@@ -27,18 +27,19 @@ function render_block_core_template_part( $attributes ) {
 		$template_part_id    = $attributes['theme'] . '//' . $attributes['slug'];
 		$template_part_query = new WP_Query(
 			array(
-				'post_type'      => 'wp_template_part',
-				'post_status'    => 'publish',
-				'post_name__in'  => array( $attributes['slug'] ),
-				'tax_query'      => array(
+				'post_type'           => 'wp_template_part',
+				'post_status'         => 'publish',
+				'post_name__in'       => array( $attributes['slug'] ),
+				'tax_query'           => array(
 					array(
 						'taxonomy' => 'wp_theme',
 						'field'    => 'name',
 						'terms'    => $attributes['theme'],
 					),
 				),
-				'posts_per_page' => 1,
-				'no_found_rows'  => true,
+				'posts_per_page'      => 1,
+				'no_found_rows'       => true,
+				'lazy_load_term_meta' => false,
```

Comment:
See https://github.com/WordPress/gutenberg/pull/47999#pullrequestreview-1297199708, can you please add an inline comment above this line on why this is good for performance?

Reply:
@spacedmonkey Because it's less obvious why this argument is needed / useful here than the other ones.

Reply:
Yes I had the same need in previous PRs of this type, basically safeguard the future people from removing micro optimisations like this or from not being aware of them.

Reply:
Added comment.